src/ConfigSetup.cpp

```diff
@@ -96,6 +96,8 @@ ConfigSetup::ConfigSetup(void)
   sys.moves.intraMemc = DBL_MAX;
   out.state.settings.enable = true;
   out.restart.settings.enable = true;
+  out.state_dcd.settings.enable = true;
```

Comment:
It seems like it is `true` by default. What happens if the user doesn't specify the DCD frequency? Would this situation leave `out.state_dcd.settings.enable` to `true` and leave the frequency at `ULONG_MAX`?

Reply:
@YounesN, good point. It follows the same logic as `out.restart` and `out.state`. It would be an issue if user does not specify these parameters but getting an error like "Error: Coordinate frequency is not specified!". I put more error checking in config file, and also I disabled the default value for `out.restart`, `out.state`, `out.state_dcd`, and `out.restart_dcd`.